.gitignore

```diff
@@ -184,3 +184,4 @@ lcov.info
 /keyman*.changes
 /keyman*.tar.?z
 
+build/
```

Comment:
We cannot add this globally -- `resources/build` is tracked, as is `docs/build`, as are many test fixture `build` folders

```suggestion
```

(Generally, I am very wary of making top-level .gitignore changes because there are just too many moving parts and it's hard to see all the ramifications)

Reply:
Turns out if we move es-bundling to `web/src/tools` then the build directory is already ignored, so we don't need this change.